bin/minifollowups/pycbc_foreground_minifollowup

```diff
@@ -108,26 +108,32 @@ if args.sort_variable not in f[file_val]:
                    '{1} are {2}'.format(args.sort_variable, file_val,
                                         ', '.join(all_datasets)))
 
+# In case we are doing background minifollowup with repeated events,
+# we must include the ordering / template / trigger / time info for
+# _many_ more events to make sure we get enough
 events_to_read = num_events * 100
 
+# Hold your horses! - we've asked for more events than there are!
 if len(stat) < num_events:
     num_events = len(stat)
 if len(stat) < events_to_read:
     events_to_read = len(stat)
 
+# Get the indices of the events we are considering in the order specified
 sorting = f[file_val + '/' + args.sort_variable][:].argsort()
 if args.sort_order == 'descending':
     sorting = sorting[::-1]
 event_idx = sorting[0:events_to_read]
 stat = stat[event_idx]
 
+# Save the time / trigger / template ids for the events
 times = {}
 tids = {}
-# Version used for multi-ifo coinc code
 ifo_list = f.attrs['ifos'].split(' ')
 for ifo in ifo_list:
     times[ifo] = f['{}/{}/time'.format(file_val,ifo)][:][event_idx]
     tids[ifo] = f['{}/{}/trigger_id'.format(file_val, ifo)][:][event_idx]
+bank_ids = f['{}/template_id'.format(file_val)][:][event_idx]
```

Comment:
```suggestion
bank_ids = f[file_val]['template_id'][:][event_idx]
```
though you can remove quite a bit of verbosity with an auxiliary variable `f_cat = f[file_val]` around line 102.